ModelCatalogueCorePluginTestApp/test/functional/org/modelcatalogue/core/remoteTesting/FinalizedDataModelIsMarkedAsFinalizedInXMLSpec.groovy

```diff
@@ -23,6 +24,10 @@ import spock.lang.Title
 class FinalizedDataModelIsMarkedAsFinalizedInXMLSpec extends GebSpec {
 
     def 'Examine that finalized data model is marked as finalized in the XML'() {
```

Comment:
@Vendetta30 Could you annotate this test with: 

`@IgnoreIf({ !System.getProperty("downloadFilepath") })`